### ISSUE TITLE
Error: The method 'FlushbarRoute.install' has more required arguments than those of overridden method 'OverlayRoute.install'.

### ISSUE THREAD

**Issue by anasmuhamad98** [AndreHaueisen/flushbar#117]
flutter run
Using hardware rendering with device sdk gphone x86. If you get graphics artifacts, consider enabling software rendering with "--enable-software-rendering".
Launching lib\main.dart on sdk gphone x86 in debug mode...
Running Gradle task 'assembleDebug'...

Compiler message:

../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flushbar-1.9.1/lib/flushbar_route.dart:273:8: Error: The method 'FlushbarRoute.install' has more required
arguments than those of overridden method 'OverlayRoute.install'.

  void install(OverlayEntry insertionPoint) {

       ^

/C:/flutter/packages/flutter/lib/src/widgets/routes.dart:40:8: Context: This is the overridden method ('install').

  void install() {

       ^

../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flushbar-1.9.1/lib/flushbar_route.dart:281:18: Error: Too many positional arguments: 0 allowed, but 1 found.

Try removing the extra positional arguments.

    super.install(insertionPoint);

                 ^

FAILURE: Build failed with an exception.

* What went wrong:

A problem was found with the configuration of task ':app:compileFlutterBuildDebug'.

> Cannot write to file 'C:\Users\anasmuhamad98\AndroidStudioProjects\umtouch' specified for property 'outputFiles' as it is a directory.

* Try:

Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 45s
Running Gradle task 'assembleDebug'...                             48.7s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by thomasmot** [comment]
Same problem here :/

**Comment by motrieux-thomas** [comment]
Hi, this is a recent issue that appeared with the latest version of flutter.
The signature of the install method has been changed.

**SOLUTION :** 
Downgrade your version (stable ?)
**OR**
Edit the flushbar's files : "flushbar-1.9.1/lib/flushbar_route.dart"

_Line 273_
void install() {

_Line 281_
super.install();

Have a nice day :)

**Comment by aburcheri** [comment]
Have the same problem, will it be fixed or what has to be done? With 

>  flushbar:
      git:
         url: https://github.com/AndreHaueisen/flushbar.git
         ref: 13c55a8
it works on Android but not on IOS

**Comment by jakobkuehne** [comment]
Auf jedem Gerät mit der aktuellen Flutter Beta (benötigt für Flutter Web?) kommt der Fehler. Mit der letzten Flutter Stable Version nicht. Ich hoffe es wird bald behoben.

**Comment by jakobkuehne** [comment]
On any device with the current Flutter Beta (required for Flutter Web?) the error occurs. Not with the latest Flutter Stable version. I hope it will be fixed soon. 

Thank you thomasmot for the temporary fix. Works perfectly, but of course can't be a solution forever.

**Comment by crtl** [comment]
```yaml
# pubspec.yaml
flushbar:
    git:
      url: https://github.com/AndreHaueisen/flushbar
      ref: master
```

works.

**Comment by IFang5** [comment]
In my case the suggested edit to pubspec.yaml caused a version collision with one of the dependencies.
I just added a reference to a stable old version of flushbar in the dependencies section in pubspec.yaml

dependencies:
    flushbar: 1.9.1

**Comment by anasmuhamad98** [comment]
Thank you all

**Comment by omar-khaium** [comment]
try `flushbar: 1.6.0`
It workes for me. Also, check the changelogs for the compatible version for you.

**Comment by biklas7** [comment]
> try `flushbar: 1.6.0`
> It workes for me. Also, check the changelogs for the compatible version for you.

This was fixed on the latest version `1.10.0`. Check the changelog
